src/common/com/intellij/plugins/haxe/util/HaxePresentableUtil.java

```diff
@@ -58,12 +59,12 @@ public static String unwrapCommentDelimiters(@NotNull String text) {
   }
 
   @NotNull
-  public static String getPresentableParameterList(HaxeNamedComponent element) {
+  public static String getPresentableParameterList(PsiElement element) {
```

Comment:
Why are you making these more generic?

Reply:
Because of inconsistency of BNF rules. Parameters list could be defined not only in HaxeNamedComponent (look at HaxeEnumValueModel).

Reply:
Wait a minute.  An enumValue extends (via mixin) HaxePsiField, which extends HaxeComponent, which extends HaxeNamedComponent.  It must have been something else causing the issue.

At any rate, PsiElement is very low level thing and seems overly broad.

Reply:
Ahha, I've just changed a bit parsing rule for enum value, and reverted parameter type for `getPresentableParameterList`